### PR TITLE
Add new test and fix for 3309

### PR DESCRIPTION
**Pull request by willwendorfvm**
Test that conf_remap can override float settings (e.g. proxy.config.http.background_fill_completed_threshold) successfully

### CODE REVIEW

File: tests/gold_tests/autest-site/trafficserver.test.ext [hunk 1]
```diff
@@ -221,6 +221,11 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True, enabl
     tmpname = os.path.join(config_dir, fname)
     p.Disk.File(tmpname, id=make_id(fname), typename="ats:config")
 
+    # for describe-float, used by conf_remap plugin
+    fname = "mysterio.config"
```
Comment (SolidWallOfCode) [inline tests/gold_tests/autest-site/trafficserver.test.ext]: Can't use that name, it's internal. Use something like "delain.config". "Delain" is *always* a good choice for names in examples.

File: tests/gold_tests/remap/conf_remap_float.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import os
+
+Test.Summary = '''
+Test command: sudo traffic_ctl config describe proxy.config.http.background_fill_completed_threshold (YTSATS-3309)
```
Comment (SolidWallOfCode) [inline tests/gold_tests/remap/conf_remap_float.py]: It should not be necessary to `sudo` if TS is started up as a normal user.  Susan also says that `config describe` shouldn't need `sudo` in any case.

**Review by SolidWallOfCode — COMMENTED** [review]
Looks good with a few changes.

File: tests/gold_tests/autest-site/trafficserver.test.ext (hunk 1 above)
Comment (willwendorfvm) [reply tests/gold_tests/autest-site/trafficserver.test.ext]: Changed to delain.

File: tests/gold_tests/remap/conf_remap_float.py (hunk 2 above)
Comment (willwendorfvm) [reply tests/gold_tests/remap/conf_remap_float.py]: Removed sudo. I think that was a typo.

**Comment by SolidWallOfCode** [comment]
Why did you merge this in to your master?

**Comment by willwendorfvm** [comment]
Thought it was normal when closing. The PR back to the open source repo is still off the branch, though.

**Pull request merged**